src/components/home/MapsContainer.vue

```diff
@@ -57,19 +57,36 @@
                     type="area"
                 />
             </section>
+            <v-container v-if="areasFiltered.length === 0">
+                <v-alert
+                    variant="outlined"
+                    type="warning"
+                    border="left"
+                >
+                  There are no areas with this name
```

Comment:
You should use translation key for i18n. GeoGuess is translated in more than 10 languages. 
Add you key in https://github.com/GeoGuess/GeoGuess/blob/master/src/lang/locale/en.json
and use $t in the component
```
{{ $t('Home.Sections.xxx') }}

````